templates/base_index.html

```diff
@@ -101,7 +101,15 @@ <h2 class="p-heading--three u-no-max-width">
     <section class="p-strip is-deep u-image-position is-bordered">
       <div class="row">
         <div class="col-7">
-          <img src="https://assets.ubuntu.com/v1/0bfbd13b-cloud_section_bundle.svg?h=460" class="u-image-position--top u-hide--small" alt="" height="460" style="height: 100%; right: 47%;" />
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/450d7c2f-openstack-hero.svg",
+              alt="",
```

Comment:
Why blank alt?

Reply:
Just matched the current alt. Also its not that useful for accessibility. Might be worth removing altogether in the future.

Reply:
If you think they're all decorative, fair enough but don't remove the alt as that's against WebAIM recommendations. https://webaim.org/techniques/alttext/